xbmc/AppParamParser.cpp

```diff
@@ -40,7 +41,7 @@ void CAppParamParser::Parse(const char* const* argv, int nArgs)
 void CAppParamParser::DisplayVersion()
 {
   printf("%s Media Center %s\n", CSysInfo::GetVersion().c_str(), CSysInfo::GetAppName().c_str());
-  printf("Copyright (C) 2005-2013 Team %s - http://kodi.tv\n", CSysInfo::GetAppName().c_str());
+  printf("Copyright (C) %s Team %s - http://kodi.tv\n", CSysInfo::GetAppName().c_str(), CCompileInfo::GetCopyrightYears());
```

Comment:
Values are now swapped. Should be the other way around

Reply:
that was stupid..